### PR TITLE
fix(FEC-10733): embedPlaykit service doesn't throw critical error when confvars are not present

### PR DESCRIPTION
**Pull request by dan-ziv**
We need to call dieError instead of dieGracefully to signal a critical error and return 404 so it won't be cached.

### CODE REVIEW

File: alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php [hunk 1]
```diff
@@ -651,7 +651,7 @@ private function initMembers()
 		//Get bundle configuration stored in conf_vars
 		$confVars = $this->uiConf->getConfVars();
 		if (!$confVars) {
-			KExternalErrors::dieGracefully("Missing bundle configuration in uiConf, uiConfID: $this->uiconfId");
+			KExternalErrors::dieError("Missing bundle configuration in uiConf, uiConfID: $this->uiconfId");
```
Comment (OrenMe) [inline alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php:654]: please add the error ENUM `KExternalErrors::BUNDLE_CREATION_FAILED`

**Comment by MosheMaorKaltura** [comment]
reviewed

File: alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php (hunk 1 above)
Comment (dan-ziv) [reply alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php]: done

File: alpha/apps/kaltura/lib/KExternalErrors.class.php [hunk 2]
```diff
@@ -119,7 +120,8 @@ class KExternalErrors
 		self::LIVE_STREAM_CONFIG_NOT_FOUND => "Live stream playback config not found for requested live entry",
 		self::TOO_MANY_PROCESSES => "Too many executed processes",
 		self::BUNDLE_CREATION_FAILED => "Failed to build bundle for [%s]",
-		self::ENTRY_NOT_SEQUENCE => "One or more of the sequence entry ids given is not a sequence entry",
+        self::MISSING_BUNDLE_CONFIGURATION => "Missing bundle configuration in uiConf, uiConfID: [%s]",
+        self::ENTRY_NOT_SEQUENCE => "One or more of the sequence entry ids given is not a sequence entry",
```
Comment (OrenMe) [inline alpha/apps/kaltura/lib/KExternalErrors.class.php:124]: please fix alignment

File: alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php [hunk 3]
```diff
@@ -651,7 +651,7 @@ private function initMembers()
 		//Get bundle configuration stored in conf_vars
 		$confVars = $this->uiConf->getConfVars();
 		if (!$confVars) {
-			KExternalErrors::dieGracefully("Missing bundle configuration in uiConf, uiConfID: $this->uiconfId");
+            KExternalErrors::dieError(KExternalErrors::MISSING_BUNDLE_CONFIGURATION, "" . $this->uiconfId);
```
Comment (OrenMe) [inline alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php:654]: please fix alignment

File: alpha/apps/kaltura/lib/KExternalErrors.class.php (hunk 2 above)
Comment (dan-ziv) [reply alpha/apps/kaltura/lib/KExternalErrors.class.php]: They don't have prettier... :)
done

File: alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php (hunk 3 above)
Comment (dan-ziv) [reply alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php]: done

**Review by OrenMe — APPROVED** [review]

**Comment by MosheMaorKaltura** [comment]
reviewed

**Pull request merged**